code/__DEFINES/atmospherics/atmos_mapping_helpers.dm

```diff
@@ -24,10 +24,10 @@
 #define ATMOS_TANK_HALON "halon=100000;TEMP=293.15"
 #define ATMOS_TANK_HEALIUM "healium=100000;TEMP=293.15"
 #define ATMOS_TANK_H2 "hydrogen=100000;TEMP=293.15"
-#define ATMOS_TANK_HYPERNOBLIUM "nob=100000;TEMP=293.15"
+#define ATMOS_TANK_HYPERNOBLIUM "hypernoblium=100000;TEMP=293.15"
 #define ATMOS_TANK_MIASMA "miasma=100000;TEMP=293.15"
 #define ATMOS_TANK_NITRIUM "nitrium=100000;TEMP=293.15"
-#define ATMOS_TANK_PLUOXIUM "pluox=100000;TEMP=293.15"
+#define ATMOS_TANK_PLUOXIUM "pluoxium=100000;TEMP=293.15"
```

Comment:
shouldn't these all be using your defines?

Reply:
I'd like them to, yea